d2core/d2ui/button.go

```diff
@@ -40,16 +40,24 @@ const (
 	ButtonTypeSquareClose        ButtonType = 20
 	ButtonTypeSquareOk           ButtonType = 21
 	ButtonTypeSkillTreeTab       ButtonType = 22
-	ButtonTypeMinipanelOpenClose ButtonType = 23
-	ButtonTypeMinipanelParty     ButtonType = 24
-	ButtonTypeBuy                ButtonType = 25
-	ButtonTypeSell               ButtonType = 26
-	ButtonTypeRepair             ButtonType = 27
-	ButtonTypeRepairAll          ButtonType = 28
-	ButtonTypeLeftArrow          ButtonType = 29
-	ButtonTypeRightArrow         ButtonType = 30
-	ButtonTypeQuery              ButtonType = 31
-	ButtonTypeSquelchChat        ButtonType = 32
+	ButtonTypeQuestDescr         ButtonType = 23 // I suppose it should be also used in WP tabs
```

Comment:
the waypoint tab constants can just be an alias to these constants later, i think this is okay.